### DISCUSSION TITLE
New async Functions - Nameing...

### DISCUSSION THREAD

**Discussion by jogibear9988** [quartznet/quartznet#567] [General] [answered]
Should the new await-able Functions wich return a Task have "Async" added to their Name?

see: https://docs.microsoft.com/en-us/dotnet/csharp/async
"You should add "Async" as the suffix of every async method name you write."

**Comment by lahma** [comment, marked as answer]
Copying my reply from https://github.com/quartznet/quartznet/issues/441 :

> I think that discussion has been on the table enough in the .NET OSS community. Here's [NServiceBus's take on the matter](https://docs.particular.net/nservicebus/upgrades/5to6/async-suffix).

So naming will be kept as-is, public API cannot be changed to other direction that easily.
